andromeda/inaturalist.py

```diff
@@ -52,17 +53,41 @@ def add_fieldnames(self, new_fieldnames):
         self.fieldnames.extend(new_fieldnames)
 
 
-def get_inaturalist_observations(user_id, add_sat_rgb_data, add_landcover_data):
+def count_observerations(user_id):
+    observations_dict = pyinaturalist.get_observations(user_id=user_id, count_only=True)
+    return observations_dict["total_results"]
+
+
+def get_observations(user_id, limit=None):
+    total_observations = count_observerations(user_id=user_id)
+    if limit:
+        # when using per_page the observations are sorted with the most recent first
+        observations_dict = pyinaturalist.get_observations(user_id=user_id, per_page=limit)
+    else:
+        # when using per_page the observations are sorted descending
```

Comment:
This comment seems out of place since `per_page` was used above and this uses `page`? Isn't the difference just in how much to return and defaulting the way it's returned?